opentelemetry-sdk/src/opentelemetry/sdk/metrics/__init__.py

```diff
@@ -241,6 +241,7 @@ def __init__(
         description: str,
         unit: str,
         value_type: Type[metrics_api.ValueT],
+        meter: "Accumulator",
```

Comment:
was the existing behaviour of leaving the meter out of the observer a missed implementation detail or was it intentional?

Reply:
I _think_ it is the first, all the instruments are produced by a specific `Meter` method, so it is perfectly possible to have it in `Observer`s as well.

Reply:
The original reason why `meter` was added as a reference in the instruments themselves was because the instruments needed a way to access the  corresponding `view_data` that controlled their aggregation. Since observers did not utilize views, this was not necessary for the to have a reference to the meter.